action-token-authenticator/src/test/java/org/keycloak/quickstart/ArquillianActionTokenWithAuthenticatorTest.java

```diff
@@ -71,18 +73,26 @@
 import static org.keycloak.test.TestsHelper.keycloakBaseUrl;
 
 @RunWith(Arquillian.class)
+@Ignore("This one sets ExternalAppAuthenticator as required but on the other hand needs a user. This way we have a clash - a user can not in using forms since this guy is required and then this guy can not get a user. ")
```

Comment:
Don't ignore tests - leave it in and we can fix this test in a separate PR

Reply:
Resolving, see https://github.com/keycloak/keycloak-quickstarts/pull/166#discussion_r380576280